common/loki.tf

```diff
@@ -0,0 +1,19 @@
+resource "helm_release" "loki" {
+  name             = "loki"
+  namespace        = "loki"
+  create_namespace = true
+
+  repository = "https://grafana.github.io/helm-charts"
+  chart      = "loki"
+  version    = var.loki_version
+
```

Comment:
I think the Loki/Promtail should be optional. If we start to put to much thing in Kubic this will become to heavy.

What do you think @p-bizouard ?


And use this Terraform best practice to enable or not some feature
``` hcl
count = var.enable_log_aggregate ? 1 : 0 
```

Reply:
You are right, I will update the PR

Reply:
@claudusd I updated the MR, providing the variable "loki_enabled" valued to `false` by default.